src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -1136,6 +1138,19 @@ private void CheckAllowedProjectChanges(ProjectChanges projectChanges)
             }
         }
 
+        private bool CanApplyParseOptionChange(ParseOptions old, ParseOptions @new, Project project)
```

Comment:
Rather than resorting to verbatim names, could we just use names like `oldOptions` and `newOptions`?